packages/dhis2/src/Adaptor.js

```diff
@@ -508,33 +565,52 @@ export function upsert(
   return state => {
     console.log(`Preparing upsert via 'get' then 'create' OR 'update'...`);
```

Comment:
This log line doesn't really make sense on the new tracker - let's move it into the `else`